packages/linear-progress/src/mwc-linear-progress.ts

```diff
@@ -77,16 +77,16 @@ export class LinearProgress extends BaseElement {
   @observer(function(this: LinearProgress, value: boolean) {
     this.mdcFoundation.setReverse(value);
   })
-  reverse = 0;
+  reverse = false;
```

Comment:
Let's make all the `type: Boolean` properties also `reflect: true`. This is the rule on which we've settled.